appscale/tools/appscale.py

```diff
@@ -279,9 +291,19 @@ def up(self):
     # Construct a run-instances command from the file's contents
     command = []
     for key, value in contents_as_yaml.items():
+      if key in self.REMOVED_ASF_ARGS:
+        raise BadConfigurationException("'{}' is no longer in use. Please refer "
+                                        "to our  website for more "
```

Comment:
Do we have a more specific URL? I wouldn't know where to look on our website. Also, extra space after "our".